armi/reactor/reactorParameters.py

```diff
@@ -210,13 +215,13 @@ def defineCoreParameters():
 
         pb.defParam(
             "critSearchSlope",
-            units="1/day",
+            units=f"1/{units.DAYS}",
             description="Critical keff search slope",
         )
 
         pb.defParam(
             "doublingTime",
-            units="EFPY",
+            units=units.YEARS,
```

Comment:
Similar comment as above about preserving the "EFP" part in the description.

If someone were to think this were purely "years", they would get a very wrong answer.

Reply:
Fixed